packages/ui/src/components/ContentRenderer.tsx

```diff
@@ -793,7 +804,7 @@ export default function ContentRenderer({
   isEdited = false,
   viewMode = 'chat',
 }: {
-  post: Post;
+  post: Post | { type: 'chat' | 'diary' | 'gallery'; id: string; content: any };
```

Comment:
What type error does this fix? Seems like the added type would already be a valid post?